rftp/msgs.go

```diff
@@ -47,11 +47,27 @@ type option struct {
 }
 
 func (o *option) UnmarshalBinary(data []byte) error {
-	panic("not implemented") // TODO: Implement
+	if len(data) < 2 {
+		return fmt.Errorf("Option too short")
+	}
+
+	o.otype = data[0]
+	o.length = data[1]
+	o.value = data[2:]
+
+	if int(o.length) != len(data)-2 {
```

Comment:
This is only called from a function in which data is sliced according to `o.length`, thus this will never be false.